typescript/functions.ts

```
@@ -268,57 +268,48 @@ function getPersonNames(people: Person[]): string[] {
 // Exception handling
 function demonstrateExceptionHandling(): void {
     try {
-        throw new Error("This is a test exception");
+        const result: string = "This should be a number";
+        return result;
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)